src/Stepic/RobotDemo.java

```diff
@@ -0,0 +1,98 @@
+package Stepic;
+//3.3 the task is to make the robot move to the desired coordinate
+
+ class RobotDemo {
+     public static void main(String[] args) {
+         Robot robot = new Robot(50,-30, Direction.DOWN);
+         moveRobot(robot, -10, 20);
+     }
+
+     public enum Direction {
+         UP,
+         DOWN,
+         LEFT,
+         RIGHT
+     }
+
+     public static class Robot {
+             int x;
```

Comment:
I don't understand what does it mean - 'x' and what should it do? 
What is the value of this variable?
This is the coordinateX or something else?